src/lib/src/select-container.spec.ts

```diff
@@ -39,7 +39,12 @@ describe('SelectContainerComponent', () => {
   });
 
   it('should not throw when clicking the element immediately on creation', () => {
-    const selectContainer = fixture.debugElement.query(By.directive(SelectContainerComponent)).nativeElement;
-    expect(() => triggerDomEvent('mousedown', selectContainer)).not.toThrowError();
+    const selectContainer = fixture.debugElement.query(By.directive(SelectContainerComponent));
+    expect(() => triggerDomEvent('mousedown', selectContainer.nativeElement)).not.toThrowError();
+  });
+
+  it('should expose update as part of the public api', () => {
+    const selectContainer = fixture.debugElement.query(By.directive(SelectContainerComponent));
+    expect(typeof selectContainer.componentInstance.update).toEqual('function');
```

Comment:
How about we also test if `calculateBoundingClientRect` was called on each item to verify that the logic works? Maybe we could also add a spy on `calculateBoundingClientRect` and verify that it has been called. What do you think?